gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip
```

Comment:
I was thinking of bumping Gradle to latest version (6.2.2 IIRC). It should be safe to use the latest Gradle version since we have test that check against older versions.

Reply:
I actually did in previous commits. My reasoning was that Gradle metadata support officially become stable in 6.0. But I had an issue with Checkstyle task and reverted.